Nagstamon/Servers/__init__.py

```diff
@@ -25,12 +25,18 @@
 
 from collections import OrderedDict
 
+# we need this for uncommon modules
+try:
+    import icinga2api
+    icinga2api_is_available = True
+except:
+    pass
```

Comment:
Here would be a good place to set icinga2api_is_available = False, otherwise it will crash later, right?